spec/scrabble_spec.rb

```diff
@@ -0,0 +1,69 @@
+require './lib/scrabble'
+
+describe Scrabble::Scrabble do
+
+  describe "This method returns the score of a given word" do
```

Comment:
Quick note on naming conventions in rspec: Generally, the `describe` blocks should contain the name of the method itself. So in this case, you'd say `describe ".score" do`.